backend/dataall/base/utils/api_handler_utils.py

```diff
@@ -24,13 +24,22 @@
 ]
 ENGINE = get_engine(envname=ENVNAME)
 ALLOWED_ORIGINS = os.getenv('ALLOWED_ORIGINS', '*')
+AWS_REGION = os.getenv('AWS_REGION')
 
 
 def redact_creds(event):
```

Comment:
nit: That's why I don't like this approach, you have to remember to obfuscate stuff